packages/mdc-textfield/_mixins.scss

```diff
@@ -157,14 +157,12 @@
 }
 
 @mixin mdc-text-field-dense_ {
+  // NOTE: This is an eyeball'd approximation of what's in the mocks.
+  @include mdc-text-field-label-float_(110%, 0%, $mdc-text-field-dense-label-scale);
```

Comment:
Did you not give 110% a var name since its only used here?

Reply:
Ya...I think there's probably some bugs around the floating label with a dense text field...but I decided to keep this PR as a pure refactor